doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -483,9 +516,18 @@ Libraries for networking
 
   * Added an error code :c:enumerator:`FOTA_DOWNLOAD_ERROR_CAUSE_INTERNAL` to indicate that the source of error is not network related.
 
+* :ref:`lib_nrf_cloud_rest` library:
+
+  * Replaced ``nrf_cloud_rest_cell_pos_get`` function with :c:func:`nrf_cloud_rest_location_get` function.
+
 * Added:
 
   * :ref:`lib_lwm2m_location_assistance` library that has support for using A-GPS, P-GPS and ground fix assistance from nRF Cloud using an LwM2M server.
+  * :ref:`lib_nrf_cloud_location` library that provides support for device location from nRF Cloud using cellular and/or WiFi network information.
+
+* Removed:
```

Comment:
Let's rethink this entry. Maybe it needs to be under :ref:`lib_nrf_cloud_cell_pos` library and say that the library is deprecated, the new library to be used is the nRF Cloud location one. And also state that it will be removed from the documentation in the NCS release 2.3.0.

Reply:
So, we keep the current lib_nrf_cloud_cell_pos library doc in the release, but have a comment in the beginning of it saying that it is deprecated and the new lib_nrf_cloud_location library is replacing it. Also mention, that in the next NCS release, this library will be removed.

In the release notes, have a separate entry for it saying that it is deprecated. The same story that is in the actual deprecated library doc file

Reply:
but the library itself no longer exists... isn't that misleading to say it is deprecated?
the functions and source files referenced in the .rst have been removed

Reply:
Oh, that changes everything. I thought it is still there, but it is going to be replaced by the new one. Then remove the old lib document altogether. And use the tag of the removed file in addition to the one you have in the new doc (to avoid broken links). Now that you remove the old lib, in the changelog, just mention it as a changed (name changed) library, if there are no actual functional changes. Note that you cannot anymore link to the removed file, so just write the actual name of the old lib.